src/views/dashboard/Dashboard.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { useEffect, useState } from "react";
-import DarkMode from "../_shared/DarkMode";
```

Comment:
Yeah, thank you for moving this. We can create a userSettings component and throw this into that.